### ISSUE TITLE
Added a default keybinding for Jupyter:Run Selection/Line of ctrl+alt+enter breaks Replace All shortcut

### ISSUE THREAD

**Issue by franciscocpg** [DonJayamanne/pythonVSCode#470]
After `Added a default keybinding for Jupyter:Run Selectiong/Line of ctrl+alt+enter` on release 0.5.0 ctrl+alt+enter shortcut (replace all action) is not working in vscode.

Here is the output from developer tools
```
Conflict detected, command `editor.action.replaceAll` cannot be triggered by ctrl+alt+enter due to jupyter.runSelectionLine (keybindingResolver.ts:190)
```

**keybindingResolver.ts:190**
```
if (KeybindingResolver.whenIsEntirelyIncluded(true, conflict.when, item.when)) {
				// `item` completely overwrites `conflict`
				if (this._shouldWarnOnConflict && item.isDefault) {
					console.warn('Conflict detected, command `' + conflict.commandId + '` cannot be triggered by ' + Keybinding.toUserSettingsLabel(keypress) + ' due to ' + item.command);
				}
				this._lookupMapUnreachable[conflict.commandId] = this._lookupMapUnreachable[conflict.commandId] || [];
				this._lookupMapUnreachable[conflict.commandId].push(conflict.keybinding);
			} 
```

**Comment by franciscocpg** [comment]
System info
```
Visual studio code

Version 1.6.1
Commit 9e4e44c19e393803e2b05fe2323cf4ed7e36880e
Date 2016-10-13T15:43:46.054Z
Shell 1.3.7
Renderer 52.0.2743.82
Node 6.5.0
```
```
XUbuntu

NAME="Ubuntu"
VERSION="16.04.1 LTS (Xenial Xerus)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 16.04.1 LTS"
VERSION_ID="16.04"
HOME_URL="http://www.ubuntu.com/"
SUPPORT_URL="http://help.ubuntu.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
UBUNTU_CODENAME=xenial

Linux francisco-Inspiron-5537 4.4.0-45-generic #66-Ubuntu SMP Wed Oct 19 14:12:37 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by franciscocpg** [comment]
If anyone else is having the same problem and don't want to uninstall the plugin, my workaround was to override ctrl+alt+enter shortcut at `~/.config/Code/User/keybindings.json` with default replace all action shortcuts.
```json
    {
        "key": "ctrl+alt+enter",
        "command": "editor.action.replaceAll",
        "when": "editorFocus && findWidgetVisible"
    },
    {
        "key": "ctrl+alt+enter",
        "command": "search.action.replaceAll",
        "when": "replaceActive && searchViewletVisible && !findWidgetVisible"
    }
```

**Comment by gandhis1** [comment]
I fixed this in PR #423. When you have a search or find dialog open, Jupyter will not react to the command.
